src/main/menu_app/src/components/Navbar.js

```diff
@@ -1,25 +1,24 @@
 import React from "react";
-import { NavLink } from "react-router-dom";
+import { Link } from "react-router-dom";
 
 import './../App.css';
 
 function Navbar() {
   return (
-  <div className='container'>
-    <nav className='navbar'>
-      <ul className='navList'>
-        <li>
-          <NavLink href="/" className='navItem'>Home</NavLink>
+    <nav className='navbar bg-body-tertiary px-3 mb-3'>
```

Comment:
nit: I think all these HTML tags should be moved 2 spaces to the left, no? Or it was wrong before?

Reply:
![image](https://github.com/user-attachments/assets/becf2441-fad9-4016-8f35-ecd6eae48bcb)
I fixed previously missed formatting mistake along with rewriting the component.

Reply:
ok